compiler+runtime/test/cpp/jank/read/parse.cpp

```diff
@@ -63,6 +63,23 @@ namespace jank::read::parse
       CHECK(r.expect_ok().unwrap().end == r.expect_ok().unwrap().start);
     }
 
+    TEST_CASE("Ratio") {
+      SUBCASE("Single Ratio") {
```

Comment:
Let's add a subcase for `1/0`, which should fail. I just tested this in a Clojure repl and it fails. We should detect this while parsing, not while lexing.

```
user=> 1/0
Syntax error reading source at (REPL:27:0).
Divide by zero
```